### PR TITLE
Added process button for unprocessed files

### PR DESCRIPTION
**Pull request by robouden**
Add button for forced processing

### CODE REVIEW

**Comment by robouden** [comment]
replaces 241

File: app/controllers/bgeigie_imports_controller.rb [hunk 1]
```diff
@@ -39,6 +39,12 @@ def fixdrive
     @bgeigie_import.fixdrive!
     redirect_to @bgeigie_import
   end
+  
+  def process_button
+    @bgeigie_import = scope.find(params[:id])
+    @bgeigie_import.process_button!
```
Comment (matschaffer) [inline app/controllers/bgeigie_imports_controller.rb]: Have you tried just calling `.process` here? Seems like the `.process_button!` method you've added is exactly the same as `.process`

File: app/models/bgeigie_import.rb [hunk 2]
```diff
@@ -71,6 +71,14 @@ def process
     delete_tmp_file
   end
 
+def process_button!
```
Comment (matschaffer) [inline app/models/bgeigie_import.rb]: If `.process` works, we can remove this method.

File: app/controllers/bgeigie_imports_controller.rb (hunk 1 above)
Comment (robouden) [inline app/controllers/bgeigie_imports_controller.rb]: Yes.. tried that first but I created errors.. I think i can fix that..

File: app/models/bgeigie_import.rb (hunk 2 above)
Comment (robouden) [inline app/models/bgeigie_import.rb]: Originally I tried.. But it did not work.. not sure why..

**Pull request merged**